lib/events/events_test.go

```diff
@@ -987,3 +995,145 @@ func TestEvents(t *testing.T) {
 		})
 	}
 }
+
+var (
```

Comment:
nit: make this a const?

Reply:
Also looks like we can actually move them inside `getDefaultValue()` where they are used?

Reply:
I can't make `eventString` a const and don't want to move it into `getDefaultValue` as that would run `strings.Repeat` every time `getDefaultValue` is called which seems wasteful